src/model_signing/signature/pki.py

```diff
@@ -147,28 +158,30 @@ def verify(self, bundle: bundle_pb.Bundle) -> None:
             )
 
         store_ctx = ssl_crypto.X509StoreContext(
-            self._store, signing_cert_ossl, chain)
+            self._store, signing_cert_ossl, chain
+        )
         try:
             store_ctx.verify_certificate()
         except ssl_crypto.X509StoreContextError as err:
             raise VerificationError(
-                f'signing certificate verification failed: {err}'
+                f"signing certificate verification failed: {err}"
             ) from err
         usage = signing_cert_crypto.extensions.get_extension_for_class(
-            x509.KeyUsage)
+            x509.KeyUsage
+        )
         if not usage.value.digital_signature:
             raise VerificationError(
-                ('the certificate is not valid for digital'
-                 ' signature usage'))
+                ("the certificate is not valid for digital" " signature usage")
+            )
```

Comment:
this auto format looks weird now that it fits all on one line

Reply:
Good catch. I'll fix this